scene/2d/cpu_particles_2d.cpp

```diff
@@ -1035,14 +1132,29 @@ void CPUParticles2D::_set_redraw(bool p_redraw) {
 void CPUParticles2D::_update_render_thread() {
 	if (OS::get_singleton()->is_update_pending(true)) {
 		update_mutex.lock();
-		VS::get_singleton()->multimesh_set_as_bulk_array(multimesh, particle_data);
+		if (_interpolated) {
+			VS::get_singleton()->multimesh_set_as_bulk_array_interpolated(multimesh, particle_data, particle_data_prev);
+		} else {
+			VS::get_singleton()->multimesh_set_as_bulk_array(multimesh, particle_data);
+		}
 		update_mutex.unlock();
 	}
 }
 
 void CPUParticles2D::_notification(int p_what) {
 	if (p_what == NOTIFICATION_ENTER_TREE) {
 		set_process_internal(emitting);
+
+		// For interpolated version to update the particles right away,
+		// we need a sequence of events.
+		// First ensure we are in _interpolated mode if the Node is set to interpolated.
+		_refresh_interpolation_state();
+
+		// Now, if we are interpolating, we want to force a single tick update.
+		// If we don't do this, it may be an entire tick before the first update happens.
+		if (_interpolated) {
+			_update_internal(true);
```

Comment:
The delta for the priming here could possibly be tweaked, depending on feedback from users. This assumes the particle system is entering the tree on a physics tick, and won't get an update till the next physics tick.

If it enters on a frame rather than tick, there could be a slight inconsistency, and we could possibly detect this and use the `physics_interpolation_fraction` to estimate a sensible delta.

But would be fine for a follow up PR based on feedback.